src/pages/ServiceDetails/ServiceInfo/ServiceInfoRouteRules/RouteRuleRoute.tsx

```diff
@@ -10,7 +10,7 @@ import './RouteRuleRoute.css';
 interface RouteRuleRouteProps {
   name: string;
   route: DestinationWeight[];
-  validations: Map<string, ObjectValidation>;
+  validations: { [k: string]: ObjectValidation };
```

Comment:
Same here, a "key" I think it helps to not miss we are coding a map. I see "key" is used in other files, too.